README.jp.md

```diff
@@ -0,0 +1,87 @@
+# Evals 
+
+Evalsは、OpenAIモデルを評価するためのフレームワークであり、ベンチマークのオープンソースレジストリでもあります。
+
+Evalsを使用して、以下のような評価を作成し、実行することができます。
+- プロンプトを生成するためにデータセットを使用する。
+- OpenAIモデルが提供する補完の品質を測定し
+- 異なるデータセットやモデル間で性能を比較することができます。
+
+Evalsでは、できるだけシンプルに、できるだけ少ないコードでevalを構築できるようにすることを目指しています。まずは、以下のステップを**順番に踏むことをお勧めします。
+1. このドキュメントを読み、[以下の設定方法](README.md#Setup)に従います。
+2. 既存のevalを実行する方法を学びます。[run-evals.md](docs/run-evals.md).
+3. 既存のevalのテンプレートに慣れる。[eval-templates.md](docs/eval-templates.md).
+4. evalを作成するプロセスを手順に沿って行う。eval-templates.md (docs/eval-templates.md)
```

Comment:
`eval-templates.md (docs/eval-templates.md)` should be changed to `[build-eval.md](docs/build-eval.md)`.